README.md

```diff
@@ -60,35 +57,53 @@ If you **have a predictive problem**, you can **submit it as a data challenge**
 
 ## How to use this bundle?
 
-Start by installing ramp-workflow (this library):
+Start by pip-installing `ramp-workflow` (this library):
 
 ```bash
-git clone https://github.com/paris-saclay-cds/ramp-workflow.git
-cd ramp-workflow
-python setup.py install
+$ pip install git+https://github.com/paris-saclay-cds/ramp-workflow.git
 ```
 
+This will set up some command line scripts like `ramp_test_submission`.
+We suggest to use a dedicated virtual environment if you are familiar with it.
+
 ### Get familiar with starting kits
 
-Starting kits in [ramp-kits][rkits] are working workflows and workflow instantiations. They work out of the box. You can run them using the `ramp_test_submission` script that simply executes [`testing_py`](rampwf/utils/testing.py) in the starting kit. For example, clone the titanic starting kit and test it by
+Starting kits in [ramp-kits][rkits] are working workflows and workflow instantiations. 
+They work out of the box. 
+
+Start by cloning the starting kit `<ramp_kit_name>` in the repository of your choosing
+
+```bash
+$ cd /path/to/your/ramp-kits/repo
```

Comment:
I would replace 'repo' with eg 'directory' as the ramp-kits is not a repo, you are cloning a repo with the next step